data/blog/electron-11-0.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Electron 11.0.0
+author:
+- VerteDinde
+date: '2020-11-17'
+---
+
+Electron 11.0.0 has been released! It includes upgrades to Chromium `87`, V8 `8.7`, and Node.js `12.18.3`. We've added support for Apple silicon, and general improvements. Read below for more details!
+
+---
+
+The Electron team is excited to announce the release of Electron 11.0.0! You can install it with npm via `npm install electron@latest` or download it from our [releases website](https://electronjs.org/releases/stable). The release is packed with upgrades, fixes, and new support for Apple's M1 hardware. 
+
+We can't wait to see what you build with them! Continue reading for details about this release, and please share any feedback you have!
+
+## Notable Changes
+
+### Stack Changes
+
+* Chromium `87.0.4280.47`
+    * [New in Chrome 86](https://developers.google.com/web/updates/2020/10/nic86)
+    * [New in Chrome 87]()
+* Node.js `12.18.3`
+    * [Node 12.18.3 blog post](https://nodejs.org/en/blog/release/v12.18.3/)
+    * [Node 12.7.0 blog post](https://nodejs.org/en/blog/release/v12.17.0/)
+* V8 `8.7`
+    * [V8 8.6 blog post](https://v8.dev/blog/v8-release-86)
+    * [V8 8.7 blog post](https://v8.dev/blog/v8-release-87)
+
+### Highlight Features
+
+* Support for Apple Silicon: On November 10, Apple announced their [new M1 chips, which will be included in their upcoming hardware](https://www.apple.com/newsroom/2020/11/apple-unleashes-m1/). As of Electron 11, Electron will be shipping separate versions of Electron for Intel Macs (x64) and Apple's upcoming M1 hardware (arm64). You can learn more about how to get your Electron app [running on Apple's M1 hardware here.](https://www.electronjs.org/blog/apple-silicon) [#24545](https://github.com/electron/electron/pull/24545)
+* Added V8 crash message and location information to crashReport parameters. [#24771](https://github.com/electron/electron/pull/24771)
+* Fixed memory leak on macOS when using `dialog.showMessageBox` API. [#26099](https://github.com/electron/electron/pull/26099)
```

Comment:
This is already fixed in 8, 9, and 10 so I'm unsure if it should be in the notes as a highlight?